_pages/clients/scala.md

```diff
@@ -202,12 +202,26 @@ try {
 }
 {% endhighlight %}
 
+
 This will get your file as a string, saving it to the variable `input`.  If the file was binary (an image, etc), you could instead use the function `.getBytes()` to retrieve the file's content as a byte array. For more image-manipulation tutorials, see the [Computer Vision Recipes]({{site.baseurl}}/tutorials/recipes/#computer-vision).
 
 Now you've seen how to upload a local data file, check if a file exists in a data collection, and download the file contents.
 
 For more methods on how to get a file using the Data API from a data collection go to the [API Specification](http://docs.algorithmia.com/#getting-a-file).
 
+{% if site.enterprise %}
+## Publishing Algorithmia Insights
+
+Inference-related metrics (a feature of [Algorithmia Insights](../algorithmia-enterprise/algorithmia-insights)) can be reported via using the `report_insights` method of the Algorithmia client.
+
+{% highlight scala %}
+# Report Algorithmia Insights
```

Comment:
<img width="803" alt="Screen Shot 2020-10-15 at 4 32 25 PM" src="https://user-images.githubusercontent.com/8980868/96196666-d1424880-0f04-11eb-8f5d-fa2ab25acab0.png">

This h1 is being rendered inside the code block, which looks a bit odd. Let's change it to `### Report Algorithmia Insights` (so it renders as an h3) and move it above the `highlight scala` line.

Reply:
One more update here.